RELEASING.md

```diff
@@ -120,3 +123,45 @@ github repository.
     $ git push upstream v$MAJOR.$MINOR.$PATCH
     $ git push upstream v$MAJOR.$MINOR.x
     ```
+
+## Deployment
+
+Deployment to Maven Central (or the snapshot repo) is for all of the artifacts
+from the project.
+
+### Branch
+
+Before building/deploying, be sure to switch to the appropriate branch or tag.
+For the current release use:
+
+```bash
+$ git checkout -b v$MAJOR.$MINOR.$PATCH
```

Comment:
Actually it worked correctly for me (maybe by luck) https://git-scm.com/docs/git-checkout. So I had one remote branch with that name (which was the tag) so it created a local branch equivalent with the remote branch.

So it depends a bit how people setup their git repo. I have mine set with origin (my fork of the project) and upstream (the main remote main instrumentation-java). If my way to use the repo is the normal one the correct command is:
git checkout -b v$MAJOR.$MINOR.$PATCH --track remote/v$MAJOR.$MINOR.$PATCH

Do you want me to modify to use this command?